userspace/libscap/scap_procs.c

```diff
@@ -378,6 +378,7 @@ int32_t scap_proc_fill_cgroups(scap_t *handle, struct scap_threadinfo* tinfo, co
 		char* subsys_list;
 		char* cgroup;
 		char* scratch;
+		char default_subsys_list[] = "cpu,memory,cpuset";
```

Comment:
The list is chosen based on what kinds of cgroups we inspect, right? It could use a comment for future readers of this code.